awx/main/tasks/host_indirect.py

```diff
@@ -45,22 +45,30 @@ def build_indirect_host_data(job: Job, job_event_queries: dict[str, dict[str, st
     facts_missing_logged = False
     unhashable_facts_logged = False
 
+    job_event_queries_no_module = {}
```

Comment:
why is it called `job_event_queries_no_module`? The loop below requires the key to be composed of three parts, wouldn't that always include a module name?

Reply:
nevermind, got it now. `parts[0:2]` does *not* include index 2 (the module name)